Project.toml

```diff
@@ -5,18 +5,23 @@ version = "2.1.1"
 
 [deps]
 Cthulhu = "f68482b8-f384-11e8-15f7-abe071a5a75f"
+FlameGraphs = "08572546-2f56-4bcf-ba4e-bab62c3a3f89"
+LeftChildRightSiblingTrees = "1d6d02ad-be62-4b6b-8a6d-2f90e265016e"
```

Comment:
To minimize [compat] bumps, how about getting this from FlameGraphs instead of making it an independent dependency?
```julia
import FlameGraphs
using FlameGraphs: Node, addchild
```
below.